Userland/Libraries/LibCore/Event.h

```diff
@@ -53,7 +53,7 @@ class DeferredInvocationEvent : public Event {
 
 public:
     DeferredInvocationEvent(NonnullRefPtr<DeferredInvocationContext> context, Function<void()> invokee)
-        : Event(Event::Type::DeferredInvoke)
+        : Event(Event::DeferredInvoke)
```

Comment:
`Event::Type` should be an enum class instead, make that change instead if you want to improve this